packages/@adobe/spectrum-css-temp/components/modal/index.css

```diff
@@ -20,7 +20,7 @@ governing permissions and limitations under the License.
   --spectrum-dialog-exit-animation-delay: 0ms;
 
   /* Distance between top and bottom of dialog and edge of window for fullscreen dialog */
-  --spectrum-dialog-fullscreen-margin: 32px;
+  --spectrum-dialog-fullscreen-margin: 40px;
```

Comment:
Is there a common place for `components/dialog/index.css` and this file to share this variable?

Reply:
Ideally I would've included it into the css var but Larry said that file was autogenerated and to put new vars at the top of the index.css file.